lib/avo/app/fields/field_extensions/visible_in_different_views.rb

```diff
@@ -15,14 +15,22 @@ def initialize(id, **args, &block)
         end
 
         def show_on(*where)
-          normalize_views(where).flatten.each do |view|
-            show_on_view view
+          if where.include? :all
+            show_on_all
+          else
+            normalize_views(where).flatten.each do |view|
+              show_on_view view
+            end
```

Comment:
Does this look more readable to you?

```suggestion
          return show_on_all if where.include? :all
          
          normalize_views(where).flatten.each do |view|
            show_on_view view
          end
```